client/src/visualization/authServerService.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+const baseUrl = 'http://localhost:3001';
+
+export interface Credentials {
+  username: string;
+  password: string;
+}
+
+export interface Admin {
+  token: string;
+  username: string;
+}
+
+const login = async (credentials: Credentials) => {
```

Comment:
Looks like login and sign will return Promise<any>, maybe we could provide the return types here, to help with type safety?